manifests/config.pp

```diff
@@ -238,4 +239,47 @@
     }
   }
 
+  file { dirname($pid):
+    ensure  => directory,
+    owner   => $setuid,
+    group   => $setgid,
+    mode    => '0644',
+    seluser => 'system_u',
+    selrole => 'object_r',
+    seltype => 'stunnel_var_run_t',
+  }
+
+
+  # These templates need variables, that's why they are here
+  if 'systemd' in $facts['init_systems'] {
+    file { '/etc/systemd/system/stunnel.service':
+      ensure  => file,
+      content => template('stunnel/connection_systemd.erb'),
+      owner   => 'root',
+      group   => 'root',
+      mode    => '0644',
+      notify  => Exec['stunnel daemon reload']
+    }
+    exec { 'stunnel daemon reload':
```

Comment:
If the service hasn't been started, what does the reload do?

Reply:
It informs systemd that there is a new until file present